public/service-worker.js

```diff
@@ -0,0 +1,40 @@
+// https://deanhume.com/create-a-really-really-simple-offline-page-using-service-workers/
+'use strict';
+
+var cacheVersion = 1;
+var currentCache = {
```

Comment:
Any reason these are `var` when we're using `const` on line 8?

Reply:
Should be safe to use all ES2015+ features in the service worker file (assuming we wrap appropriately using something like native JavaScript modules). All major browsers that support service workers also support native JavaScript modules, so it's perfectly fine to serve this code to any browsers (older browsers will just ignore it).

Reply:
My point is more that these should all be `const`, rather than a mixture of `var` and `const`.

Reply:
I agree

Reply:
Turns out we can't use `const` in GOV.UK:
```
[docker] Uglifier::Error: Unexpected token: keyword (const). To use ES6 syntax, harmony mode must be enabled with Uglifier.new(:harmony => true).
```